tests/kernel_performance/hanaperf_postinstall.pm

```diff
@@ -0,0 +1,153 @@
+# SUSE's openQA tests
+#
+# Copyright 2025 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+#
+# Summary: Post installation for HANA perf SLE16 testing
+# Inherit from install_qatestset.pm
+# Maintainer: QE-SAP <qe-sap@suse.de>
+
+package hanaperf_postinstall;
+use base 'y2_installbase';
+use power_action_utils 'power_action';
+use strict;
+use warnings;
+use utils;
+use testapi;
+use Utils::Architectures;
+use repo_tools 'add_qa_head_repo';
+use mmapi 'wait_for_children';
+use version_utils qw(is_sle);
+use ipmi_backend_utils;
+
+sub install_sleperf {
+    my $sleperf_source = get_var('SLE_SOURCE');
+    my $ver_path = "/root";
+
+    # Download SLEperf package, extract and install
+    assert_script_run("wget --quiet -O $ver_path/sleperf.tar $sleperf_source 2>&1");
+    assert_script_run("tar xf /root/sleperf.tar -C /root");
+    assert_script_run("cd /root/sleperf/SLEPerf; ./installer.sh scheduler-service");
+    assert_script_run("cd /root/sleperf/SLEPerf; ./installer.sh common-infra");
+}
+
+sub extract_settings_qaset_config {
+    my $values = shift;
+    my @fields = split(/;/, $values);
+    if (scalar @fields > 0) {
+        foreach my $a_value (@fields) {
+            record_info(${a_value});
+            assert_script_run("echo '${a_value}' >> /root/qaset/config");
+        }
+    }
+}
+
+sub setup_environment {
+    my $qaset_role = get_var('QASET_ROLE', 'HANA');
+    my $mitigation_switch = get_var('MITIGATION_SWITCH', 'mitigations=auto');
+    my $qaset_kernel_tag = get_var('QASET_KERNEL_TAG', '');
+    my $ver_cfg = get_var('VER_CFG');
+    my $disable_services = "qaperf.service chronyd.service firewalld.service";
+
+    # Fill $ver_cfg by default value if it is undefined
+    unless ($ver_cfg) {
+        my $mybuild = check_var('BUILD', 'GM') ? "GM" : "Build" . get_var("BUILD", '');
+        $ver_cfg = "PRODUCT_RELEASE=SLES-" . get_var('VERSION') . ";PRODUCT_BUILD=$mybuild";
+        record_info($ver_cfg);
+    }
+
+    # Disable service
+    assert_script_run("systemctl disable $disable_services");
+
+    # sync time
+    assert_script_run("chronyd -q 'server 0.europe.pool.ntp.org iburst'");
+
+    # set static hostname
+    assert_script_run("hostnamectl hostname `hostname -s`");
+
+    # create basic /root/qaset/config
+    assert_script_run("mkdir -p /root/qaset/");
```

Comment:
Same suggestion in these 2 lines regarding double and single quotes.

Reply:
thanks! fixed.